test/microphysics_noneq_tests.jl

```diff
@@ -66,25 +66,25 @@ function test_microphysics_noneq(FT)
         pᵥ_si = TD.saturation_vapor_pressure(tps, T, TD.Ice())
         qᵥ_si = TD.q_vap_saturation_from_density(tps, T, ρ, pᵥ_si)
 
-        qₚ(qᵥ) = TD.PhasePartition(FT(qᵥ))
+        #qₚ(qᵥ) = TD.PhasePartition(FT(qᵥ))
```

Comment:
Could we keep both the new one and `PhasePartition` one in tests? Or if you want to have one, then I would keep the `PhasePartition` one. Since internally it should be calling the new version